Source/Fuse.Reactive.JavaScript/Observable.uno

```diff
@@ -143,6 +143,11 @@ namespace Fuse.Reactive
 			return new Subscription(this, observer);
 		}
 
+		IDisposable IObservableArray.Subscribe(IObserver observer)
```

Comment:
See the docs. It returns `ISubscription` if the implementation supports mutating the source, a plain `IDisposable` otherwise.